src/CMakeLists.txt

```diff
@@ -18,7 +17,6 @@ set(path_include_common
 ${FMT_INCLUDE_DIR}
 ${RDKAFKA_INCLUDE_DIR}
 ${FLATBUFFERS_INCLUDE_DIR}
-${RAPIDJSON_INCLUDE_DIR}
 ${CURL_INCLUDE_DIRS}
```

Comment:
Do we need JSONFMC++ adding to this?

Reply:
As far as I can see, there is no Find module for cmake.